other-docs/guides/getting-help-with-altis.md

```diff
@@ -1,50 +1,91 @@
+To open a support request, first you'll have to [login to the Altis Dashboard](dashboard.altis-dxp.com/).
 
-We break down support into different types. Each has their own language and processes, but they're somewhat intuitive once you internalize the language a little. Let's go over the types.
+From there, you'll find the link to the support page in the top right corner. See below.
 
-<img width="1282" alt="Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+![Altis Dashboard support link](./assets/altis-support-button.png)
+
+From this screen you'll be able to view your open issues, all previous issues, and you can also query open and previous issues if you're looking for something specific. 
+
+To open a new support ticket, simply click the blue 'Create New Ticket' button.
+
+## Create Support Ticket
+
+A quick overview of the of the screen: 
+
+**Application**
+
+Select the or all Application(s)/Environment(s) to which the support ticket concerns.
+
+**Type**
+
+The type of support ticket, of which there are currently 4. Question, Incident, Problem, and Task. We'll go into more detail about which these categories below, but most tickets will generally fall into one of these four. 
+
+**Priority**
+
+You can change the priority of a support issue, if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.
+
+**Subject**
+
+Short line describing the support request.
+
+**Description**
+
+The body of the support request.
+
+**CCs**
+
+You can CC additional relevant parties to the support request if desired. They will recieve carbon copies of the correspondance.
+
+## Support Ticket Types
+
+We break down support into different categories. Each has their own language and processes, but they should be somewhat intuitive. Let's go over the types.
+
+![Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu](./assets/altis-support-type.png)
 
 We currently have four types of support. Below is a description of each and how they are different from each other.
 
 ### Incident
 
-Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an Incident, and requires investigation.
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
```

Comment:
```suggestion
Incidents are unplanned events that negatively impact service, or the quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
```